src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/step/close/AttemptCloseStep.kt

```diff
@@ -35,11 +37,10 @@ class AttemptCloseStep(
 
     override fun getUpdatedManagedIndexMetaData(currentMetaData: ManagedIndexMetaData): ManagedIndexMetaData {
         return currentMetaData.copy(
-            step = name,
-            stepStartTime = getStepStartTime().toEpochMilli(),
-            transitionTo = null,
```

Comment:
Do we not need to reset transitionTo to null anymore?

Reply:
I think changing `transistionTo` in Step is illogical since `Steps` shouldn't not need to know about `transistionTo` but since that is not the case right now I will add back in. We should refactor this to be outside of the step.